kubernetes/kubernetes_service.go

```diff
@@ -356,3 +365,39 @@ func (in *IstioClient) GetSelfSubjectAccessReview(namespace, api, resourceType s
 	}
 	return result, err
 }
+
+func (in *IstioClient) IsMixerDisabled() bool {
+	if in.isMixerDisabled != nil {
+		return *in.isMixerDisabled
+	}
+
+	cfg := config.Get()
+	istioConfig, err := in.GetConfigMap(cfg.IstioNamespace, IstioConfigmapName)
+	if err != nil {
+		log.Warningf("IsMixerDisabled: Cannot retrieve Istio ConfigMap.")
+		return true
+	}
+
+	meshConfigYaml, ok := istioConfig.Data["mesh"]
+	log.Tracef("meshConfig: %v", meshConfigYaml)
+	if !ok {
+		log.Warningf("IsMixerDisabled: Cannot find Istio mesh configuration.")
+		return true
+	}
+
+	meshConfig := istioMeshConfig{}
+	err = yaml.Unmarshal([]byte(meshConfigYaml), &meshConfig)
```

Comment:
Why don't use the json.Unmarshall in the ""encoding/json" library instead ?
Just learning here why two different libraries are used for same function.

Reply:
This is code inherited from @israel-hdez from previous PR. I had to move it into this package for better testability. Didn't refactor the method itself though. And I think I prefer not to in this PR. See it here: https://github.com/kiali/kiali/pull/2598/files#diff-87fb62602e6a5ead1f80e010eb9609deL356

And besides that, the MeshConfig file is in YAML format not in JSON. So I guess JSON.Unmarshall would fail.